### PR TITLE
Feat: Cart Management On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer Members, please review Cart Management Code
- Add To Cart
- Remove From Cart 
- Move To WIshlist
- Initialize User Cart Mechanism

https://user-images.githubusercontent.com/68817653/160963343-7e6a404b-9bf9-4048-b400-fe5f05b114bb.mp4

### CODE REVIEW

File: src/styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.no-cart-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* .no-cart-section .no-cart-img{
+    width: 50%;
+    height: auto;
+} */
```
Comment (Vishruta-Patil) [inline src/styles/cart.css]: Remove commented lines of code

File: src/styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+/* Totals section  */
+
+.totals-item {
+    clear: both;
+    width: 100%;
+    margin-bottom: 10px;
+}
+
+.price-label {
+    float: left;
```
Comment (Vishruta-Patil) [inline src/styles/cart.css:55]: Suggestion - Avoid using float

**Review by Vishruta-Patil — APPROVED** [review]
Everything else looks good to merge

**Review by VLeads — APPROVED** [review]
Your code looks good. It's easy to understand !

File: src/styles/cart.css (hunk 2 above)
Comment (Rishi05051997) [reply src/styles/cart.css:55]: Only one place I used this, on the float I got the better result that's why I used that.

File: src/styles/cart.css (hunk 1 above)
Comment (Rishi05051997) [reply src/styles/cart.css]: Yes its removed

**Pull request merged**